cmd/opentelemetry/app/exporter/elasticsearchexporter/integration_test.go

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build integration
+
+package elasticsearchexporter
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/olivere/elastic"
+	"github.com/stretchr/testify/require"
+	"github.com/uber/jaeger-lib/metrics"
+	"go.uber.org/zap"
+
+	"github.com/jaegertracing/jaeger/model"
+	"github.com/jaegertracing/jaeger/pkg/es/config"
+	eswrapper "github.com/jaegertracing/jaeger/pkg/es/wrapper"
+	"github.com/jaegertracing/jaeger/pkg/testutils"
+	"github.com/jaegertracing/jaeger/plugin/storage/es"
+	"github.com/jaegertracing/jaeger/plugin/storage/es/dependencystore"
+	"github.com/jaegertracing/jaeger/plugin/storage/es/spanstore"
+	"github.com/jaegertracing/jaeger/plugin/storage/es/spanstore/dbmodel"
+	"github.com/jaegertracing/jaeger/plugin/storage/integration"
+)
+
+const (
+	host            = "0.0.0.0"
+	queryPort       = "9200"
+	queryHostPort   = host + ":" + queryPort
+	queryURL        = "http://" + queryHostPort
+	indexPrefix     = "integration-test"
+	tagKeyDeDotChar = "@"
+	maxSpanAge      = time.Hour * 72
+)
+
+type IntegrationTest struct {
+	integration.StorageIntegration
+
+	client        *elastic.Client
+	bulkProcessor *elastic.BulkProcessor
+	logger        *zap.Logger
+}
+
+type storageWrapper struct {
+	writer *EsSpanWriter
+}
+
+func (s storageWrapper) WriteSpan(span *model.Span) error {
+	converter := dbmodel.FromDomain{}
```

Comment:
This is a workaround, the purpose of the storage test is the correctness. Although It would be better to convert from jaeger to OTEL and then use the remain interface method.